packages/web/src/shared/Outbound.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import External from 'src/icons/External'
+import { colors } from 'src/styles'
+
+export default function Outbound({ url }: { url: string }) {
+  return (
+    <a href={externalizeURL(url)} target="_blank">
+      <External size={12} color={colors.dark} />
+    </a>
+  )
+}
+
+export function externalizeURL(url: string) {
```

Comment:
what's this for?

Reply:
if i throw `www.celo.org` straight into a `href` on an anchor its going to direct the person to `celo.org/www.celo.org`.

whereas `//www.celo.org` is know to be a domain. 

this way any URLs set in the airtable will go to the right spot.